tap_hubspot/__init__.py

```diff
@@ -191,6 +200,12 @@ def load_schema(entity_name):
             "properties": custom_schema,
         }
 
+        if entity_name in ["deals"]:
+            v3_schema = get_v3_schema(entity_name)
+            for key, value in v3_schema.items():
+                if 'hs_date_entered' in key or 'hs_date_exited' in key or 'hs_time_in' in key:
```

Comment:
I'm not 100% sure, but could you use `any` to make this line a bit easier on the eyes? It's essentially saying `are any of the strings in key` right?

Reply:
No, it's checking for any of the three prefixes in `key`

Reply:
if it's a prefix why not do `startswith` (I think that's the method in python)? And why does that mean you can't use `any`?

https://stackoverflow.com/questions/3389574/check-if-multiple-strings-exist-in-another-string

Reply:
I cleaned this up with an `any()`